kobo/settings/base.py

```diff
@@ -422,9 +422,35 @@
         'Enable most recent password validation which will prevent the user from '
         'reusing the most recent password.',
     ),
+    'ENABLE_CUSTOM_PASSWORD_GUIDANCE_TEXT': (
+        False,
+        'Enable custom password guidance text to help users create their passwords.',
+    ),
+    'CUSTOM_PASSWORD_GUIDANCE_TEXT': (
+        json.dumps({
+            'default': '',
```

Comment:
Because its appearance is controlled by `ENABLE_CUSTOM_PASSWORD_GUIDANCE_TEXT`, let's have a real default based on default rules.
While #4560 is not merged, we should follow the same technic used in #4549 and make the default string translatable, i.e:
- Add the translatable string to `static_list.py`
- Copy the same string in `base.py` - without `t()`
- Create the unit tests to ensure strings match (see `test_mfa_help_text_default_translation` in #4549).
- Use the constant from `static_list.py` in environment endpoint and its unit tests